src/handler/updateVrm.ts

```diff
@@ -67,6 +68,12 @@ export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayPr
 
       await updateVehicle(recordsToArchive, recordsToUpdate);
 
+      const recordsToSend: SNSMessageBody[] = [];
+
+      recordsToUpdate.forEach((record) => recordsToSend.push({ ...record, userEmail: userDetails.email }));
```

Comment:
Again, nothing functionally wrong but could be condensed with a map:
```suggestion
const recordsToSend: SNSMessageBody[] = recordsToUpdate.map((record) => ({ ...record, userEmail: userDetails.email }));
```